src/app/pages/applications/chart-releases/chart-releases.component.ts

```diff
@@ -495,7 +493,7 @@ export class ChartReleasesComponent implements AfterViewInit, OnInit, OnDestroy
       width: '50vw',
       minWidth: '650px',
       maxWidth: '850px',
-      data: { appName: name, type: PodSelectDialogType.Shell },
+      data: { appName: name, title: 'Choose pod', type: PodSelectDialogType.Shell },
```

Comment:
Needs to be translated.

Reply:
Title translated in html template
```suggestion 
<h1 mat-dialog-title>{{ hasPool ? title : 'No Pools Found' | translate }}</h1>
```